docs/Using Price Feeds/feed-registry.html

```diff
@@ -11,28 +11,25 @@
     0: "/files/OpenGraph_V3.png"
 ---
 {% markdown %}
-> 🚧 Open Alpha
-> 
-> Chainlink Feed Registry is in alpha. We're continuing to test the feature with users and improve the experience based on your feedback. The registry contract is also undergoing a security audit.
->
-> Note that the alpha registry will be decomissioned after the alpha period and replaced with a new registry at a new address. Please do not use it in production.
 
 ![Feed Registry](/files/feed-registry.png)
 
-The Chainlink Feed Registry is an on-chain mapping of assets to feeds. It enables you to query Chainlink price feeds from a pair of asset and denomination addresses directly, without needing to know the feed contract addresses. They enable smart contracts to retrieve the latest price of an asset in a single call, from a single contract.
+The Chainlink Feed Registry is an on-chain mapping of assets to feeds. It enables you to query Chainlink price feeds from asset addresses directly, without needing to know the feed contract addresses. They enable smart contracts to retrieve the latest price of an asset in a single call, from a single contract.
+
+# Base and Quote
 
-# Assets and Denominations
+The Feed Registry maps feeds using `base` and `quote` address pairs as keys. The Feed Registry fully supports the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" target="_blank">`AggregatorV3Interface`</a> API, but with additional `base` and `quote` inputs to support multiple pairs and feeds on a single contract. 
 
```

Comment:
Please use markdown style for all urls, we have a script that opens all external links in other pages by default.